app/javascript/app/routes/app-routes/app-routes.js

```diff
@@ -192,6 +194,12 @@ export default [
     exact: true,
     headerGradient: HEADER_GRADIENTS.commitments
   },
+  FEATURE_COMMITMENTS_OVERVIEW && {
```

Comment:
The custom compare doesn't have a nav item on the menu. You access it through the compare all targets.

Reply:
I know, I added that for testing purpose, until the "compare all targets" page will be ready. Should I remove it anyway for not confusing our client?

Reply:
No, it's ok then. Thanks. We just have to remember

Reply:
ok, thank you :wink: